.github/workflows/pr-golangci-lint.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@ec5d18412c0aeab7936cb16880d708ba2a64e1ae # tag=v6.2.0
         with:
-          version: v1.60.2
+          version: v1.63.4
           args: --out-format=colored-line-number
           working-directory: ${{matrix.working-directory}}
+      - name: Lint API
+        run: GOLANGCI_LINT_EXTRA_ARGS=--out-format=colored-line-number make lint-api
```

Comment:
In theory we could add `--new-from-rev=main` to allow this to lint only changes since the main branch, which would allow us to start enforcing the linting rules for new changes to older APIs as well 🤔

Reply:
Sounds good to me